Rubberduck.Core/AutoComplete/AutoCompleteDoBlock.cs

```diff
@@ -0,0 +1,10 @@
+﻿using Rubberduck.Parsing.Grammar;
+
+namespace Rubberduck.AutoComplete
+{
+    public class AutoCompleteDoBlock : AutoCompleteBlockBase
+    {
+        public AutoCompleteDoBlock()
+            : base($"{Tokens.Do} ", Tokens.Loop) { }
```

Comment:
This likely breaks the `Do...Loop` case, but prevents `Sub DoSomething`  from being wrongly autocompleted with a stray `Loop` keyword. Removing that space and implementing compilation check in the base class would fix that.